schema-engine/cli/README.md

```diff
@@ -30,17 +34,15 @@ schema-engine-cli 0.1.0
 Doesn't start a server, but allows running specific commands against Prisma
 
 USAGE:
-    schema-engine cli --datasource <datasource> <SUBCOMMAND>
+    schema-engine cli <SUBCOMMAND>
```

Comment:
Should rather be `schema-engine --datasource <JSON> cli <SUBCOMMAND>` but unfortunately this is how `structopt` prints the generated help message, omitting the required flags before the subcommand.